ghost/values.yaml

```diff
@@ -0,0 +1,294 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+#   storageClass: myStorageClass
+
+## Bitnami Ghost image version
+## ref: https://hub.docker.com/r/bitnami/ghost/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/ghost
+  tag: 3.8.0-debian-10-r0
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
```

Comment:
you can remove this one, Okteto Cloud always overwrite this value to `PullAlways`

Reply:
ok great